docs/design/values.md

```diff
@@ -76,21 +76,27 @@ There are three expression categories in Carbon:
 Expressions in one category can be converted to any other category when needed.
 The primitive conversion steps used are:
 
--   A _read_ converts a reference expression into a value expression.
--   _Direct initialization_ converts a value expression into an initializing
+-   A <a id="read">_read_</a> converts a reference expression into a value
     expression.
--   _Copy initialization_ converts a reference expression into an initializing
-    expression.
--   _Temporary materialization_ converts an initializing expression into a
-    reference expression.
+-   <a id="direct-init">_Direct initialization_</a> converts a value expression
+    into an initializing expression.
+-   <a id="copy-init">_Copy initialization_</a> converts a reference expression
+    into an initializing expression.
+-   <a id="temporary-materialization">_Temporary materialization_</a> converts
+    an initializing expression into a reference expression.
 
 These conversion steps combine to provide the transitive conversion table:
 
-|               From: | value                   | reference | initializing     |
-| ------------------: | ----------------------- | --------- | ---------------- |
-|        to **value** | ==                      | read      | temporary + read |
-|    to **reference** | direct init + temporary | ==        | temporary        |
-| to **initializing** | direct init             | copy init | ==               |
+|               From: | value                           | reference     | initializing             |
+| ------------------: | ------------------------------- | ------------- | ------------------------ |
+|        to **value** | ==                              | [read][]      | [temporary][] + [read][] |
+|    to **reference** | [direct init][] + [temporary][] | ==            | [temporary][]            |
+| to **initializing** | [direct init][]                 | [copy init][] | ==                       |
```

Comment:
From #infra discussion, I see you're modifying this table to add links. To copy my comment there:

The list and the table are separated by one line of explanatory text ("These conversion steps combine to provide the transitive conversion table:"), so the links serve to suggest mainly that the linked content is elsewhere, making the links look broken when they have differing labels, but they don't move to different text (because the linked line is already on screen).

To summarize, I believe the prior table (no links) was actually better because the links are now going to interrupt reading.

Separately, you may also consider moving "From:" out of the corner. The other headers are meant to be read vertically, so it would be natural to read this header vertically too. For contrast, "To" is per-row and not-bolded, whereas "From" is once and bolded -- so the different labeling can be a source of confusion. (but it's hard to un-bold "From", so you'd either be looking at an html table or bolding "to" for consistency)

Reply:
I think having the "From:" right-justified is helpful to make it clear it is describing the column headers.

Reply:
The table links are gone, and I"ve also added (short) sections for the four things, and used normal linking to them from the more condensed bulleted list, and linked back to them where useful in the doc.